packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -675,49 +639,49 @@ HTTP/1.1 204 No Content
 
-> The `mfaActive` parameter is a <ApiLifecycle access="deprecated" /> feature. Use the `lastFactorVerification` attribute in conjunction with `amr` to understand if the user has performed MFA for the current session. Use the [Factors API](/docs/reference/api/factors/#list-enrolled-factors) to query the factor enrollment status for a given user.
+> The `mfaActive` parameter is a <ApiLifecycle access="deprecated" /> feature. Use the `lastFactorVerification` attribute in conjunction with `amr` to understand if the user has performed MFA for the current Session. Use the [Factors API](/docs/reference/api/factors/#list-enrolled-factors) to query the factor enrollment status for a given user.
 
-#### Optional Session Properties
+#### Optional Session properties
 
 The [Create Session](#create-session-with-session-token) operation can optionally return the following properties when requested.
 
 | Property                                      | Description                                                                                                                                                                       |
 | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| cookieToken                                   | Another one-time token which can be used to obtain a session cookie by visiting either an application's embed link or a session redirect URL.                                     |
-| cookieTokenUrl                                | URL for a a transparent 1x1 pixel image which contains a one-time session token which when visited sets the session cookie in your browser for your organization.                 |
+| cookieToken                                   | Another one-time token that is used to obtain a session cookie by visiting either an application's embed link or a Session redirect URL.                                     |
+| cookieTokenUrl                                | The URL for a transparent 1x1 pixel image that contains a one-time session token that, when visited, sets the session cookie in your browser for your organization.                 |
 
-> The `cookieToken` is a <ApiLifecycle access="deprecated" /> property. Instead, use the [Authentication API](/docs/reference/api/authn/), which supports the full user authentication pipeline and produces a `sessionToken` which can be used in this API.
+> The `cookieToken` is a <ApiLifecycle access="deprecated" /> property. Instead, use the [Authentication API](/docs/reference/api/authn/), which supports the full user authentication pipeline and produces a `sessionToken` that you can use in this API.
```

Comment:
it seems to me that the "supports the full user authentication pipeline and produces a `sessionToken` that you can use in this API" is important to the sentence...so would require "that" rather than "which"

The `cookieToken` is a <ApiLifecycle access="deprecated" /> property. Instead, use the [Authentication API](/docs/reference/api/authn/) that supports the full user authentication pipeline and produces a `sessionToken` that you can use in this API.